srcpkgs/soju/files/README.voidlinux

```diff
@@ -0,0 +1,9 @@
+The system service /etc/sv/soju runs soju as system user _soju.
+
+By default, the database will be stored in /var/db/soju/ and logs will be in
+/var/log/soju/. Both directories are owned by user _soju.
+
+To initialize the database, make sure you run sojuctl as the _soju user:
+
+$ sudo su -s /bin/sh - _soju
+$ sojuctl -config /etc/soju/config create-user <username> -admin
```

Comment:
Any reason to use `sudo su -s` instead of `sudo -u _soju -s` or just `sudo -u sojuctl ...`? Would probably make sense to also disable the shell for the `_soju` user if its not required.

Reply:
`sudo su -s` instead of `sudo -u _soju -s` is my mistake, and my bad reason for doing this instead of `sudo -u sojuctl ...` was to split the command in two, making shorter lines and the `sojuctl` call on its own. But indeed, latter is better.

> Would probably make sense to also disable the shell for the _soju user if its not required.

Isn't this the default when defining a system user from a template? That is, setting the shell to `/sbin/nologin`.

Reply:
> Isn't this the default when defining a system user from a template? 

Ah yes, was just somewhat expecting that `-s` would not work without a valid shell set, but makes sense that it works for non login scenarios.